src/model.js

```diff
@@ -336,6 +334,19 @@ module.exports = function(koop) {
         this.customSymbolizers.push(symbolizer);
     }
 
+    this.getCustomSymbolizer = function (indexConfig){
+        let customSymbolizer = undefined;
+        if(indexConfig.customSymbolizer){
+            this.customSymbolizers.forEach(symbolizer => {
+                if(symbolizer.name === indexConfig.customSymbolizer){
+                    customSymbolizer = symbolizer;
+                    return;
+                }
+            });
+        }
```

Comment:
I think this can be simplified like:

```js
this.getCustomSymbolizer = function (indexConfig = {}){
    return this.customSymbolizers.find(symbolizer => {
        return symbolizer.name === indexConfig.customSymbolizer
    });
}
```
Much less to read and avoids use of `forEach` - which we try to skip because it means you are mutating an object.

Reply:
Updated